nntrainer/include/databuffer.h

```diff
@@ -106,14 +106,14 @@ class DataBuffer {
    * @param[in] file input file stream
    * @retval    void
    */
-  virtual void updateData(BufferType type) = 0;
+  virtual void updateData(BufferType type, int &status) = 0;
```

Comment:
I think documentaion and function signature mismatches (and also for other fns)

Q. Is there any reason status is not @retval? (and also for line 312, 372)

Reply:
Self answered. Because it should be updated.